include/EVT/io/pin.hpp

```diff
@@ -0,0 +1,109 @@
+/**
+ * This contains the different pin options that are available. The pin mappings
+ * are hardware specific and determined by the platform that is being compiled
+ * for
+ */
+#ifndef _EVT_PIN_
+#define _EVT_PIN_
+
+namespace EVT::core::IO
+{
+// STM32F302R8 pin names
+#ifdef STM32F302x8
+
+    enum class Pin
+    {
+        PA_0  = 0x00,
```

Comment:
Could we add some comment or documentation pointing to the definition for the mappings in this file?

Reply:
Good point, just updated.